adoc/admin-disaster-recovery-backup.adoc

```diff
@@ -0,0 +1,181 @@
+= Backup
+This chapter describes the backup of {productname}.
+
+== Data To Backup
+1. Create backup directories on external storage.
++
+```bash
+BACKUP_DIR="CaaSP_Backup_<TIMESTAMP>"
+mkdir ${BACKUP_DIR}.
+
+# Repeat to create folders for all master nodes
+mkdir /$BACKUP_DIR}/<MASTER_NAME>_<MASTER_IP>
+```
+
+2. Copy the following files/folders into the backup directory:
+* The `skuba`: command-line binary for the running cluster, use to replace nodes from cluster.
```

Comment:
Do we need to backup `skuba` CLI?

Reply:
Good question! IMO its necessary to backup `skuba` CLI to keep the version in sync with rest of the backup data in case they need to replace for any reason. But I'm open to suggestions.

Reply:
Understood. Then we need to consider the `helm` and the coming next `velero`.